crates/storage-api/proto/dev/restate/storage/v1/domain.proto

```diff
@@ -465,6 +508,7 @@ message Timer {
     }
 
     oneof value {
+        InvocationId neo_invoke = 1;
```

Comment:
Maybe also here, let's add a comment explaining why there is a `neo_invoke` variant.